tests/plotting/test_plotter.py

```diff
@@ -394,6 +394,34 @@ def test_plotter_meshes(sphere, cube):
     assert len(pl.meshes) == 2
 
 
+def test_multi_block_color_cycler():
```

Comment:
I found another one.
```suggestion
def test_multi_block_unique_colors():
```